modules/ingestor/src/service/advisory/osv/loader.rs

```diff
@@ -104,97 +107,85 @@ impl<'g> OsvLoader<'g> {
             }
 
             for affected in &osv.affected {
-                if let Some(package) = &affected.package {
-                    let mut purls = vec![];
+                // we only process it when we have a package
+
+                let Some(package) = &affected.package else {
+                    continue;
+                };
 
-                    purls.extend(translate::to_purl(package).map(Purl::from));
+                // extract PURLs
+
+                let mut purls = vec![];
+                purls.extend(translate::to_purl(package).map(Purl::from));
```

Comment:
is this some kind of hyper specific translate to purl ?

Reply:
Yup.